app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
allow the emulator to communicate with a local app-engine run (i.e. python3 gcp/app-engine/main.py)